internal/handlers/pg/msg_explain.go

```diff
@@ -67,6 +67,14 @@ func (h *Handler) MsgExplain(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		qp.Sort = nil
 	}
 
+	// Sort requires fetching all documents and sorting them in memory unless sort pushdown is enabled,
+	// limit pushdown is not applied when sort is set without sort pushdown.
+	// Skip pushdown is not support, limit pushdown is not applied when skip is set.
```

Comment:
```suggestion
	// Skip pushdown is not supported, limit pushdown is not applied when skip is set.
```

Overall, the whole comment is hard to read. I think it will be easier to understand if we separate parameters from common words. For example, "sort" sometimes means a parameter name and sometimes "sorting". We could use `backticks` for parameter names

Reply:
See the updated and if that's better please :pray: